debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/AlgorithmPicker.java

```diff
@@ -26,8 +26,9 @@
  * @author andhall
  *
  */
-public abstract class AlgorithmPicker<T extends IAlgorithm>
+public abstract class AlgorithmPicker<T extends com.ibm.j9ddr.vm29.j9.IAlgorithm>
```

Comment:
Its not obvious why this change is required.

Reply:
In Java 8 builds, the (interim) compiler seemed to get confused about the unqualified type.

Reply:
Maybe worth a comment?

Reply:
Will do.